packages/react-components/testing/karma.conf.js

```diff
@@ -31,6 +32,10 @@ module.exports = function karmaConfig(config) {
       },
       resolve: {
         extensions: ['.js', '.jsx'],
+        alias: {
+          '@department-of-veterans-affairs/web-components/react-bindings':
+            path.resolve(__dirname, '../../web-components/'),
+        },
```

Comment:
This ensures that the tests don't have a problem with the `react-bindings` import inside of `Banner.jsx`